tests/dialects/test_prql.py

```diff
@@ -41,10 +41,18 @@ def test_prql(self):
             "FROM x SORT age",
             "SELECT * FROM x ORDER BY age",
         )
+        self.validate_identity(
+            "FROM x SORT -age",
```

Comment:
I'm not sure if this is valid PRQL syntax, IIRC you need to wrap `-age` in braces?

Reply:
Btw, it also seems that PRQL's keywords are case-sensitive. So `FROM x` is not valid, but `from x` is. You can use their [playground](https://prql-lang.org/playground/) environment to check these things:

<img width="287" alt="Screenshot 2024-04-11 at 1 39 35 PM" src="https://github.com/tobymao/sqlglot/assets/46752250/a9e469e1-5108-473e-9e93-5a9230943367">